introduction.md

```diff
@@ -0,0 +1 @@
+My name is James and I go by Codhidharma sometimes on the interweb. This is an awesome introduction.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.